master/cdev.c

```diff
@@ -63,13 +63,14 @@ static int eccdev_mmap(struct file *, struct vm_area_struct *);
 #define VM_FAULT_CHANGE_VERSION KERNEL_VERSION(4, 10, 0)
 
 #if LINUX_VERSION_CODE >= PAGE_FAULT_VERSION
-#if (LINUX_VERSION_CODE < KERNEL_VERSION(4,11,0))
-static int eccdev_vma_fault(struct vm_area_struct *, struct vm_fault *);
+#if LINUX_VERSION_CODE >= VM_FAULT_CHANGE_VERSION
+    static int eccdev_vma_fault(struct vm_fault *);
```

Comment:
Indentation in preprocessor macros are more misguiding than helpful in my opinion.

Reply:
You're right, there are inconsistencies in the indentations

Is this ok?

```
#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 15, 0))
static void legacy_timer_emu_func(struct timer_list *t)
{
    struct legacy_timer_emu *lt = from_timer(lt, t, t);
    lt->function(lt->data);
}
#endif
```